third_party/WebKit/Source/modules/webcl/WebCLOpenCL.cpp

```diff
@@ -215,15 +186,40 @@ cl_int (CL_API_CALL *web_clGetGLTextureInfo)(cl_mem, cl_gl_texture_info, size_t,
 // by this spec.
 #define MAP_FUNC_TO_WRAPPER(fn, major, minor) { *(void**)(&fn) = (void*)fn##Impl##major##minor; }
 
-static const char* defaultLibs[] = LIBS;
-static const int defaultLibsAmount = SO_LEN;
+static Vector<String> defaultLibs;
```

Comment:
Having a global static non-POD variable like this is usually a bad idea. You probably need to use `DEFINE_STATIC_LOCAL` or something similar.